t/lib/OpenQA/Test/Client.pm

```diff
@@ -0,0 +1,22 @@
+package OpenQA::Test::Client;
+use Mojo::Base -base;
+use Test::Mojo;
+use OpenQA::Client;
+
+use Exporter 'import';
+
+our @EXPORT = qw(client);
+
+# setup test application with API access
+# note: Test::Mojo looses its app when setting a new ua (see https://github.com/kraih/mojo/issues/598).
+sub client {
```

Comment:
I would have preferred adding this to the test utilities as it seems unlikely that related functions are added into this new module later. I would have also preferred a name like `create_client` for a function which actually creates a new instance and not just returns an existing one.

Reply:
yes, this is why this PR was open for 7 days and I asked for feedback.